### PR TITLE
feat(badge): new badge component

### PR DESCRIPTION
**Pull request by elenagiere**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/collab-ui/collab-ui-angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
There is no badge component for Angular 7

**What is the new behavior?**
Now there is

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -223,3 +223,15 @@ or in case of multiple issues:
 ### Angular TypeScript
 
 [Adhere to the Code Guide](https://angular.io/guide/styleguide)
+
+### Running this project locally
+* You may need to run `$ yarn watch:lib` in one tab within terminal
```
Comment (pauljeter) [inline CONTRIBUTING.md]: Can you remove the "may" from these statements? Especially the index.ts comment below.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -223,3 +223,15 @@ or in case of multiple issues:
...
+* Then in another tab, `$ yarn serve` which will serve it at localhost:4200
+
+### Using Angular CLI with this Project
+###### When adding a new Module or Component to the src/lib directory...
+* `ng g module <componentName> --project=@collab-ui/angular`
+* `ng g component <componentName> --project=@collab-ui/angular`
+* You may need to create an `index.ts` file under the new component directory (use existing index.ts as a guide)
+* Don't for get to add the new component to `src/lib/public_api.ts`
```
Comment (pauljeter) [inline CONTRIBUTING.md]: spelling "forget"

File: src/docs/app/app.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div class="docs-container">
-  <cui-button></cui-button>
+  <h1>Welcome to the Collab UI Toolkit for Angular 7</h1>
```
Comment (pauljeter) [inline src/docs/app/app.component.html]: Let's remove the "7" since we will keep this library up to date with whatever the latest version is.

File: src/docs/app/playground/playground.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PlaygroundComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```
Comment (pauljeter) [inline src/docs/app/playground/playground.component.spec.ts]: Can you add a test to make sure that the playground component is empty? We should not check in code in the playground component.

File: src/lib/badge/badge.component.ts [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+      `${(this.className && ` ${this.className}`) || ''}` +
+      ``;
+  }
+}
+
+/**
+* @name Default Badges
+* @description Default state of badge.
+
+* @category communication
```
Comment (pauljeter) [inline src/lib/badge/badge.component.ts]: @category is not needed in the new docs

File: src/lib/badge/index.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+/*
```
Comment (pauljeter) [inline src/lib/badge/index.ts]: Remove this comment. I think I just copied it from the "public_api.ts" file into the Button example file. Can you remove it there too?

File: src/lib/public_api.ts [hunk 7]
```diff
@@ -3,3 +3,4 @@
  */
 
 export * from './button';
```
Comment (pauljeter) [inline src/lib/public_api.ts]: Let's go ahead and remove Button now that we have a real Badge component.
